module/modmanager/manager.go

```diff
@@ -144,8 +151,19 @@ func (mgr *Manager) add(ctx context.Context, conf config.Module, conn *grpc.Clie
 		return nil
 	}
 
+	var moduleDataDir string
+	if mgr.moduleDataParentDirectory != "" {
+		moduleDataDir = filepath.Join(mgr.moduleDataParentDirectory, conf.Name)
+		// safety check to prevent exiting the moduleDataDirectory
+		if !strings.HasPrefix(filepath.Clean(moduleDataDir), filepath.Clean(mgr.moduleDataParentDirectory)) {
```

Comment:
[q] Is this possible?

Reply:
If the conf name contains characters like ".." (which is technically not allowed in the regex but might in at some point in the future and people won't remember to add this check).